examples/add_openshift_provider.yml

```diff
@@ -19,5 +19,25 @@
       miq_verify_ssl: false
       provider_verify_ssl: false
     register: result
+  - debug: var=result
+
+  - name: Add Openshift Containers Provider with Prometheus endpoint to ManageIQ
+    manageiq_provider:
+      name: 'Openshift02'
+      provider_type: 'openshift-origin'
+      state: 'present'
+      provider_api_hostname: 'os02.example.com'
+      provider_api_port: '8443'
+      provider_api_auth_token: '******'
+      metrics: true
+      metrics_type: 'prometheus'
+      prometheus_hostname: 'prometheus01.example.com'
+      prometheus_port: '80'
```

Comment:
So this needs 4 parameters, and could be 2 or 3.
`metrics: true` and `metrics_type: 'prometheus'` are IMO redundant.  I propose combining to `metrics: 'prometheus'`.

In theory, both could be infered from the fact you provided `prometheus_hostname:` and `prometheus_port:`, but IIRC we added separate `metrics: true` boolean to be able to specify it but omit host/port which had a sensible default?

@dkorn @enoodle what do you think?